doc/governance.rst

```diff
@@ -125,34 +137,39 @@ and the `issue tracker <https://github.com/scikit-learn/scikit-learn/issues>`_.
 Occasionally, sensitive discussion occurs on a private list.
 
 Scikit-learn uses a "consensus seeking" process for making decisions. The group
-tries to find a resolution that has no open objections among core developers.
-At any point during the discussion, any core-developer can call for a vote, which will
-conclude one month from the call for the vote. Any vote must be backed by a
-:ref:`SLEP <slep>`. If no option can gather two thirds of the votes cast, the
-decision is escalated to the TC, which in turn will use consensus seeking with
-the fallback option of a simple majority vote if no consensus can be found
-within a month. This is what we hereafter may refer to as "**the decision making
-process**".
-
-Decisions (in addition to adding core developers and TC membership as above)
+tries to find a resolution that has no open objections among core contributors.
+At any point during the discussion, any core contributor can call for a vote,
+which will conclude one month from the call for the vote. Most votes have to be
+backed by a :ref:`SLEP <slep>`. If no option can gather two thirds of the votes
+cast, the decision is escalated to the TC, which in turn will use consensus
+seeking with the fallback option of a simple majority vote if no consensus can
+be found within a month. This is what we hereafter may refer to as "**the
+decision making process**".
+
+Decisions (in addition to adding core contributors and TC membership as above)
 are made according to the following rules:
 
-* **Minor Documentation changes**, such as typo fixes, or addition / correction of a
-  sentence, but no change of the scikit-learn.org landing page or the “about”
-  page: Requires +1 by a core developer, no -1 by a core developer (lazy
-  consensus), happens on the issue or pull request page. Core developers are
-  expected to give “reasonable time” to others to give their opinion on the pull
-  request if they're not confident others would agree.
+* **Minor Documentation changes**, such as typo fixes, or addition / correction
+  of a sentence, but no change of the ``scikit-learn.org`` landing page or the
+  “about” page: Requires +1 by a maintainer, no -1 by a maintainer (lazy
+  consensus), happens on the issue or pull request page. Maintainers are
+  expected to give “reasonable time” to others to give their opinion on the
+  pull request if they're not confident others would agree.
 
 * **Code changes and major documentation changes**
-  require +1 by two core developers, no -1 by a core developer (lazy
+  require +1 by two maintainers, no -1 by a maintainer (lazy
   consensus), happens on the issue of pull-request page.
```

Comment:
It seems odd that changes to the documentation, which is a major form of communication for the project, can not be approved by someone from the communications team. I'd go even further and say that I'd not be shocked to find that major docs changes require approval from the communication team, as they are (by construction) the most experienced when it comes to communication.

Maybe a point for further discussion to keep this PR focussed.

Reply:
as I mentioned in the PR description, I'm not sure about what to do here, and happy to hear what folks think. For now I haven't touched what we had, but happy to do so.

Reply:
I always strived for adding a documentation team.

Since I would like to move forward with SLEP19, I am fine waiting for a subsequent SLEP that would propose new team(s) before making a change in this section.

NB: IMO, I miss such a team and therefore censure myself to propose contributors.

Reply:
Happy to have a sebsequent PR proposing a docs team.

Reply:
I do not think we need to specify other teams in the governance document as long as they are under one of the two voting groups. For example, NumPy have [multiple teams](https://numpy.org/teams/), but the teams are not explicitly listed in their [governance](https://numpy.org/doc/stable/dev/governance/governance.html).

The last thing we have to do in governance, is to consider votes by the Contributor Experience Team as a +1 when it is relevant to their domain. (I prefer to do this in a follow up to keep the scope of this PR small.)

Reply:
> that we resolve this thread, and discuss +1s on PRs in a separate PR

Resolve this thread and propose expanding "+1 on PRs" in separate PR after we make a decision about the current proposal.